pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -571,13 +581,51 @@ func (t *Test) buildVersion() *version.Version {
 	return t.rt.BuildVersion()
 }
 
+func (t *Test) clusterArch() vm.CPUArch {
+	if t._arch != nil {
+		return *t._arch // test-only
+	}
+
+	return t.cluster.Architecture()
```

Comment:
@srosenberg double checking: is this a sound way to get the cluster arch for local and remote clusters?

Reply:
Yep, I believe `c.arch` is expected to be non-nil for all clusters created `by defaultClusterAllocator`.